micro-deps/micro-deps/src/main/groovy/com/ofg/infrastructure/discovery/watcher/DependencyState.java

```diff
@@ -0,0 +1,6 @@
+package com.ofg.infrastructure.discovery.watcher;
+
+enum DependencyState {
```

Comment:
Just asking - I saw that in Groovy it was public. I don't remember the case where people will try to create their own dependency watchers. Will they not need this enum? Regardless of where they watcher lays?